tests/python/pants_test/repo_scripts/test_git_hooks.py

```diff
@@ -123,46 +92,46 @@ def assert_header_check(added_files, expected_excerpt):
 
       # Check that a file with an empty header fails.
       safe_file_dump(new_py_path, '')
-      assert_header_check(added_files=[], expected_excerpt="""\
-subdir/file.py: failed to parse header at all
-""")
+      assert_header_check(added_files=[], expected_excerpt=dedent("""\
+        subdir/file.py: failed to parse header at all
+        """))
```

Comment:
This can all be simplified to a normal string, which is simpler. I recommend using this idiom:
https://github.com/pantsbuild/pants/pull/7635/files#diff-577e94080857536b0297d8b80d82a423R116